tests/conftest.py

```diff
@@ -407,7 +407,7 @@ def collect_techsupport(request, duthost):
     # "function" scope
     testname = request.node.name
     if request.config.getoption("--collect_techsupport") and request.node.rep_call.failed:
-        res = duthost.shell("generate_dump")
+        res = duthost.shell("generate_dump -s yesterday")
```

Comment:
Can you make the time range configurable? When config option is not supplied, use default `-s yesterday`?

Reply:
Looks lit to make the time range configurable doesn't yield too much benefit.

 - So far we don't have any test case which requires log information for more than 1 day. In this sense, this modification doesn't affect test cases current running.
 - to make the time range shorter doesn't reduce the size of tarball too much.

How do you think?

Reply:
Sounds OK if there is no much improvement in dump tarball size if the time range is reduced to hours.